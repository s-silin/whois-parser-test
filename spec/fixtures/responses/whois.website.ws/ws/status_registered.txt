Domain Name: GOOGLE.WS
Domain ID: D865CD29234E835CE040010AAB015FFF
WHOIS Server: whois.markmonitor.com
Registrar URL: http://www.markmonitor.com/whois.asp
Updated Date: 2018-01-30T09:34:45Z
Creation Date: 2002-03-03T17:00:26Z
Registrar Registration Expiration Date: 2019-03-03T23:00:26Z
Registrar: MarkMonitor Inc.
Registrar IANA ID: 292
Registrar Abuse Contact Email: ccops@markmonitor.com
Registrar Abuse Contact Phone: 2083895740
Domain Status: clientTransferProhibited
Domain Status: clientDeleteProhibited
Domain Status: clientUpdateProhibited
Name Server: ns1.google.com
Name Server: ns2.google.com
Name Server: ns3.google.com
Name Server: ns4.google.com
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2018-07-24T02:26:20Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

