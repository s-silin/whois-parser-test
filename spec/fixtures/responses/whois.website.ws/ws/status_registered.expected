#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2002-03-03T17:00:26Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2018-01-30T09:34:45Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-03-03T23:00:26Z}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"
